Assets/MRTK/Core/Providers/BaseSpatialObserver.cs

```diff
@@ -88,13 +89,18 @@ public override void Initialize()
         /// </summary>
         public override void Reset()
         {
-            Destroy();
+            base.Reset();
+            Disable();
+            CleanupObservationsAndObserver();
```

Comment:
I think these additional changes to reset are what's causing the PR check to fail

Reply:
yep. just found that and am pushing a fix now.